src/extractor/extractor.cpp

```diff
@@ -127,8 +127,8 @@ int Extractor::run(ScriptingEnvironment &scripting_environment)
         auto extractor_callbacks = std::make_unique<ExtractorCallbacks>(extraction_containers);
 
         const osmium::io::File input_file(config.input_path.string());
-        osmium::io::Reader reader(input_file, osmium::io::read_meta::no);
-        const osmium::io::Header header = reader.header();
+        osmium::io::Reader nodeReader(input_file, osmium::osm_entity_bits::node);
```

Comment:
`read_metadata::no` still needs to be set, otherwise the reader will parse version, user id etc. on all nodes, ways, etc.